express/code/blocks/grid-marquee/grid-marquee.js

```diff
@@ -199,29 +202,53 @@ function decorateHeadline(headline) {
 
 async function makeRating(store) {
   const { replaceKey } = await import(`${getLibs()}/features/placeholders.js`);
-  const ratings = (await replaceKey('app-store-ratings', getConfig()))?.split(';') || [];
+
+  const [ratingPlaceholder,
+    starsPlaceholder,
+    playStorePlaceholder,
+    appleStorePlaeholder] = await Promise.all(
+    [
+      await replaceKey('app-store-ratings', getConfig()),
```

Comment:
replaceKey returns a promise, so you don't need to resolve them here and can pass directly to Promise.all.
```suggestion
     replaceKey('app-store-ratings', getConfig()),
```

Reply:
Thanks for the catch.